packages/@aws-cdk/cx-api/lib/assembly/cloud-assembly-api.ts

```diff
@@ -0,0 +1,56 @@
+import { AppRuntime, MetadataEntry, MissingContext } from '../cxapi';
+import { AssetMetadataEntry } from '../metadata/assets';
+
+export interface ICloudAssembly {
+  readonly directory: string;
+  readonly version: string;
+  readonly missing?: { [key: string]: MissingContext };
+  readonly runtime: AppRuntime;
+  readonly artifacts: ICloudArtifact[];
+  readonly stacks: ICloudFormationStackArtifact[];
```

Comment:
Those will also be represented in `.artifacts`, right? Why the duplication?

Reply:
Usability.